docs/sources-asciidoc/src/main/asciidoc/concept-chapter-Configuring_the_Media_Server.adoc

```diff
@@ -87,6 +89,8 @@ The *Port* parameter defines the port where the control channel will listen to f
 
 The *PoolSize* parameter defines the number of MGCP requests that Media Server can handle concurrently.
 
+The *channelBuffer* parameter definses the MGCP channel buffer size. By default it is set to 5000.
```

Comment:
Typo `definses` should read `defines`. Explain it limits the size of incoming MGCP messages that are received by the MGCP channel.

Reply:
Done